README.md

```diff
@@ -31,22 +15,64 @@ Controls for cloud service compliance afford banks no particular strategic or co
 
 ## Approach and Proposed Solution
 
-This FINOS project produces multiple artifacts (together forming one or multiple accelerators) that provide functional code that implements regulatory compliant configurations of cloud services with BDD tests to validate efficacy. The group review the artifacts for an accelerator and then gather feedback on process and content before iterating on additional services. A key part of the project's approach is to set quality standards across artifacts; members of all tiers can contribute to the project and ensure a common high level of quality is delivered and in less time. The group will also work with cloud service providers to produce more industry specific content and solutions.
+This FINOS project produces multiple artifacts for each CFI service.
+Where necessary, we have segmented the aforementioned artifacts across multiple repositories
+to ensure that they are ready for quick and easy adoption by users in highly regulated environments.
+
+1. Detailed documentation in the form of a _Service Approval Accelerator_ (SAA)
+    - All SAAs live within this repository
+1. Infrastructure as Code (IaC) that meets the specifications described in the SAA
+    - This IaC may include multiple services if necessary to properly ensure compliance of the specified service
```

Comment:
financial compliance

Reply:
My opinion... adding "financial" here would be redundant, taking up extra space & restating a concept we've made clear by this point in the docs

Reply:
I agree, I'm going to leave it as is and merge